src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} />
```

Comment:
```suggestion
      <TodoInfo todo={todo}  key={todo.id}/> 
```  i'm suggesting to add key={todo.id} on this element) 
Because every array elemet has to be unique, as i got from lesson ))